compilers/concrete-compiler/compiler/lib/ClientLib/EncryptedArguments.cpp

```diff
@@ -49,7 +49,7 @@ EncryptedArguments::pushArg(uint64_t arg, KeySet &keySet) {
   }
   if (!input.encryption.has_value()) {
     // clear scalar: just push the argument
-    preparedArgs.push_back((void *)arg);
+    ciphertextBuffers.push_back(ScalarData(arg));
```

Comment:
So `ciphertextBuffers` actually also contains cleartext arguments right ? And overall, `EncryptedArguments` is not really only holding encrypted arguments ?

Reply:
Yes sure that's a bit misleading, but as you said we will change that soon